wp-content/themes/csisnuclearnetwork/assets/_scss/base/_base.scss

```diff
@@ -41,6 +41,7 @@ body {
   padding-right: var(--container-padding);
   padding-left: 6vw;
   padding-left: var(--container-padding);
+  background: $color-postbg-200;
```

Comment:
You probably want to set this background color on the body tag instead of the `.container` so it extends all the way to the edges of the screen. If you don't need to do that, then ignore me!